manifests/c/CHIRPSoftware/CHIRP-next/20241206/CHIRPSoftware.CHIRP-next.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: CHIRPSoftware.CHIRP-next
+PackageVersion: '20241206'
+InstallerType: nullsoft
+ReleaseDate: 2024-12-06
+Installers:
+- Architecture: x64
+  InstallerUrl: https://archive.chirpmyradio.com/chirp_next/next-20241206/chirp-next-20241206-installer.exe
+  InstallerSha256: 5f84e57dd0ea99dac08aed16ec86876f00e70139

```

Comment:
> Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][0][InstallerSha256] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: CHIRPSoftware.CHIRP-next.installer.yaml Manifest Error: Failed to process field. [InstallerSha256] Manifest Error: Required field missing. [InstallerSha256]

Hi @radoslew,

This hash seems too short.

Reply:
Now it should be fine - sorry!